Sources/App/Views/PackageController/PackageShow+View.swift

```diff
@@ -176,6 +177,31 @@ extension PackageShow {
             )
         }
 
+        func packageSponsorship() -> Node<HTML.BodyContext> {
+            guard model.fundingLinks.count > 0
+            else { return .empty }
+
+            return .section(
+                .class("package-funding"),
+                .p(
+                    .text(model.repositoryOwnerName),
+                    .text(" accepts support for "),
```

Comment:
If that's a possibility we could sidestep it via `The maintainers welcome ...`

Reply:
D'oh, I completely missed that we're using the owner name right above.

Reply:
I wonder though if `The maintainers welcome ...` would work better generally.

`SwiftPackageIndex welcomes ...` sounds a bit weird and I suspect most accounts accepting sponsors are going to be orgs not real people where you'd say `<Name> welcomes ...` but rather `The <org> welcomes ...`. Then again `The Alamofire welcomes` is even weirder 🤔

I guess the point is that the only way to avoid grammatical weirdness is to control the subject by making it a constant `The maintainers` or `The authors`.

Just a thought.

Reply:
Yea I considered this and I think including the author/organisation name is best. It's interesting that you say it's likely to be organisations accepting sponsorship. I assumed it'd be the other way around and that it would be individuals rather than companies.

Especially for individuals, I think it's important to have a name there. When people sponsor, it's quite a human thing. Think how often people thank the two of us for SPI, as opposed to the SPI org generically. Having a connection with human(s) is going to be more temping than a connection with "the maintainers". I think I'd rather have some occasionally slightly wonky grammar than make this generic.

I'll pull some stats now on the breakdown of individuals/orgs because I'm curious, but I think I'm still on the side of the fence where putting the org name in is better than the generic.

Reply:
I got bored after doing 100, but 78% of those first 100 are individuals.

There are 256 accounts that are accepting sponsorship with 718 repositories between them. That's more than 10%!!